beta-documentation/returns-api-PROJECT-2776/returns.json

```diff
@@ -124,7 +101,7 @@
             "description": "Details of the return request",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/CreateReturnRequestReq"
+              "$ref": "#/definitions/CreateReturnReq"
```

Comment:
Since we dropped the word Request from the entity, can we now use Request and Response instead of Req and Resp?  It is not much but had to take a second look to understand what the abbreviations meant.

Reply:
We can yes. The reason for the abbreviation was to match the naming convention of the gRPC messages but it doesn't need to match those.

Reply:
Updated here @benpratt77 
https://github.com/bigcommerce/dev-docs/pull/60